src/LineBot/LineBot.cs

```diff
@@ -197,10 +197,10 @@ public async Task<IEnumerable<ILineEvent>> GetEvents(HttpRequest request)
 
             var eventCollection = JsonConvert.DeserializeObject<LineEventCollection>(jsonContent);
 
-            if (eventCollection == null || eventCollection.Events == null)
-                return Enumerable.Empty<ILineEvent>();
+            if (eventCollection == null || (eventCollection as ILineEvents).Events == null)
```

Comment:
Could you change this back to `if (eventCollection == null || eventCollection.Events == null)`?